django_app/redbox_app/templates/accessibility-statement.html

```diff
@@ -1,46 +1,39 @@
-{% import "macros.html" as macros %}
-{% extends "base_generic_gov.html" %}
+{% extends "base.html" %}
 
-<!--TODO: Update system url and other variables-->
-
-{% block title %}
-  <title>Accessibility statement - Redbox Copilot - GOV.UK</title>
-{% endblock %}
+{% set pageTitle = "Accessibility statement" %}
 
 {% block content %}
-  <h1 class="govuk-heading-l">Accessibility statement for Redbox Copilot Platform</h1>
-  <div class="govuk-body">
-    <p>This accessibility statement applies to Redbox Copilot - <a href="<SYSTEM_URL>" class="govuk-link"><SYSTEM_URL></a>.</p>
-    <p>This website is run by the Cabinet Office. We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>
+  <h1 class="govuk-heading-l">Accessibility statement for Redbox Copilot</h1>
+
+    <p class="govuk-body">This accessibility statement applies to Redbox Copilot.</p>
+    <p class="govuk-body">This website is run by the Cabinet Office. We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>
     <ul class="govuk-list govuk-list--bullet">
       <li>change colours, contrast levels and fonts</li>
```

Comment:
Is this first point possible? We might need to remove it

Reply:
The "change colours" etc. bit? If so, then yes, through user stylesheets and browser settings. There's also things like high contrast mode at OS level. It's one (of several) reasons why web pages are more accessible (out of the box) than PDFs.